napari/components/viewer_model.py

```diff
@@ -182,6 +182,7 @@ def __init__(self, title='napari', ndisplay=2, order=(), axis_labels=()):
         self.dims.events.order.connect(self._update_layers)
         self.dims.events.order.connect(self.reset_view)
         self.dims.events.current_step.connect(self._update_layers)
+        self.dims.events.point.connect(self._update_layers)
```

Comment:
I don't think `point` is ever set anywhere. Previously it was a derived property of `Dims` that was read.